packages/flutter/test/rendering/paragraph_test.dart

```diff
@@ -437,14 +437,13 @@ void main() {
     expect(paragraph.size.height, 26.0);
 
     // Test the sizes of nested spans.
-    final String text = testSpan.toStringDeep();
     final List<ui.TextBox> boxes = <ui.TextBox>[
-      for (int i = 0; i < text.length; ++i)
```

Comment:
Why hardcode to be 4? Should it use textSpan.toPlainText instead?

Reply:
The span tree looks pretty straightforward and we use equals(4) below anyway. If you prefer `toPlainText` I can switch to that?

Reply:
Yes, let's switch back to use toPlainText. If this hardcode to 4 the below `equals(4)` expect is obsolete I think.